paymentsheet/src/test/java/com/stripe/android/paymentsheet/SavedPaymentMethodsPage.kt

```diff
@@ -17,7 +17,7 @@ import com.stripe.android.paymentsheet.ui.TEST_TAG_REMOVE_BADGE
 
 class SavedPaymentMethodsPage(private val composeTestRule: ComposeTestRule) {
     fun waitUntilVisible() {
-        composeTestRule.waitUntil {
+        composeTestRule.waitUntil(timeoutMillis = 5_000) {
```

Comment:
Default is 1 second which is sometimes too small, increasing also stabilized one of the detach tests that waited until this screen was visible again.